example/spring-data-jpa/src/test/kotlin/com/linecorp/kotlinjdsl/example/spring/data/jpa/jpql/select/SelectExample.kt

```diff
@@ -51,6 +56,24 @@ class SelectExample : WithAssertions {
         assertThat(actual).isEqualTo(1L)
     }
 
+    @Test
+    fun `books reference specific book publisher entity`() {
+        // when
+        val actual = bookPublisherRepository.findAll {
+            select(
+                path(Book::isbn),
+            ).from(
+                entity(BookPublisher::class),
+                join(entity(Book::class)).on(path(BookPublisher::book).eq(entity(Book::class))),
+            ).where(
+                path(BookPublisher::publisherId).eq(3),
+            )
+        }
```

Comment:
I'm looking at Book as the Aggregation Root, and it's Book's PK that you want to query in this example, so I'd like you to remove BookPublisherRepository and write the example using BookRepository.

Reply:
I understand, let's delete the BookPublisherRepository and modify it to use BookRepository to implement the example.